components/schemas/environments/services/lb/telemetry/LoadBalancerTelemetryUrlMetrics.yml

```diff
@@ -14,15 +14,57 @@ properties:
           nullable: true
           type: object
           required:
+            - host
+            - method
+            - path
             - total
+            - last_hit
+            - handlers
           properties:
+            host:
+              type: string
+              description: The host name of the URL request
+              example: abc.cycle.io
+            method:
+              type: string
+              description: The method of the UrRL request
+              example: GET
+            path:
+              type: string
+              description: The path name of the URL request
+              example: /pathname
             total:
               type: integer
+              description: The number of requests to a certain URL
+              example: 11
+            last_hit:
+              description: The timestamp of the most recent hit to the URL
+              $ref: ../../../../Duration.yml
+            handlers:
+              type: object
+              description: An object describing the relative breakdown of proxy, cache, forward, and redirect URL responses
+              required:
+                - proxy
+                - cache
+                - forward
+                - redirect
+              properties:
+                proxy:
+                  $ref: ./LoadBalancerTelemetryUrlRequestHandler.yml
+                cache:
+                  $ref: ./LoadBalancerTelemetryUrlRequestHandler.yml
+                forward:
+                  $ref: ./LoadBalancerTelemetryUrlRequestHandler.yml
+                redirect:
+                  $ref: ./LoadBalancerTelemetryUrlRequestHandler.yml
             responses:
               type: object
+              description: An object where the key is the response type and the key is the number of hits with that response
+              example: "{ http: 404: 10 }"
```

Comment:
this example seems wrong, that's not a valid object. can you clean up the example to be accurate?